elastic/apm-server
# processor/otel: set default RepresentativeCount

## Motivation/summary

When translating OpenTelemetry spans to transactions/spans,
set RepresentativeCount to 1. This is incorrect when sampling
is enabled, but it's better to have incorrectly scaled metrics
than none at all, which is what happens when RepresentativeCount
is zero.

Until/unless some tag along the lines of Jaeger's sampler.param
is added, users will have to mentally scale down the metrics
in proportion to the sampling rate, to find the actual event counts.

## Checklist


## How to test these changes

1. Run an OpenTelemetry-instrumented application which sends HTTP requests to another service
2. Configure to send data to APM Server, capture some requests and wait a minute or so (the aggregation interval).
3. Check the service map looks right
4. Check the service overview dependencies list has the expected entries

## Related issues

Closes https://github.com/elastic/apm-server/issues/4972
